lib/jsmidgen.js

```diff
@@ -393,8 +393,15 @@ var Midi = {};
 		var trackCount = this.tracks.length.toString(16);
 
 		// prepare the file header
-		var bytes = File.HDR_CHUNKID + File.HDR_CHUNK_SIZE + File.HDR_TYPE0;
+		var bytes = File.HDR_CHUNKID + File.HDR_CHUNK_SIZE;
 
+		// set Midi type based on number of tracks
+		if (parseInt(trackCount) > 1) {
```

Comment:
Please take a look at this http://mido.readthedocs.org/en/latest/midi_files.html#file-types 

Also, while working in a desktop application for MIDI it's normal to have tracks being independent of each other (unless I ve mistaken that for something else)